kotlin-analysis-core/src/main/kotlin/org/jetbrains/research/ml/kotlinAnalysis/util/Utils.kt

```diff
@@ -27,3 +30,16 @@ fun getSubdirectories(path: Path): List<Path> {
 fun getPrintWriter(directory: Path, fileName: String): PrintWriter {
     return File(directory.toFile(), fileName).printWriter()
 }
+
+/**
+ * Returns the path to the file, containing given ktElement.
+ * Path is relative from the ktElement's project directory. )
+ */
+fun getRelativePathToKtElement(ktElement: KtElement): Path {
```

Comment:
Maybe this function is more applicable for PSI util? And I prefer extension function...Do you?